tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -131,26 +131,33 @@ Test Files 1 failed (1)
 Duration <DURATION>"
 `;
 
-exports[`bring_local_variables.w 1`] = `
-"error: Preflight field \\"x\\" is not initialized
-   --> ../../../examples/tests/invalid/file_with_variables.w:10:3
-   |
-10 |   x: num;
-   |   ^ Preflight field \\"x\\" is not initialized
+exports[`bring_local_variables.main.w 1`] = `
+"error: Module files cannot have statements besides classes, interfaces, enums, and structs. Rename the file to end with \`.main.w\` to make this an entrypoint file.
+  --> ../../../examples/tests/invalid/file_with_variables.w:5:1
+  |
+5 | let x = 5;
+  | ^^^^^^^^^^ Module files cannot have statements besides classes, interfaces, enums, and structs. Rename the file to end with \`.main.w\` to make this an entrypoint file.
 
 
-error: Cannot bring \\"../../../examples/tests/invalid/file_with_variables.w\\" - modules with statements besides classes, interfaces, enums, and structs cannot be brought
-  --> ../../../examples/tests/invalid/bring_local_variables.w:1:1
+error: Module files cannot have statements besides classes, interfaces, enums, and structs. Rename the file to end with \`.main.w\` to make this an entrypoint file.
+  --> ../../../examples/tests/invalid/file_with_variables.w:6:1
   |
-1 | bring \\"./file_with_variables.w\\" as stuff;
-  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Cannot bring \\"../../../examples/tests/invalid/file_with_variables.w\\" - modules with statements besides classes, interfaces, enums, and structs cannot be brought
+6 | let y = [\\"hello\\", \\"world\\"];
+  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^ Module files cannot have statements besides classes, interfaces, enums, and structs. Rename the file to end with \`.main.w\` to make this an entrypoint file.
```

Comment:
This is the main new error message introduced in this PR. Note that the error message is now on the module file, instead of on the file that brings the module file